applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/settings.js

```diff
@@ -0,0 +1,199 @@
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'roam_trigger_snr', _('roam_trigger_snr'), _('Minimum signal-to-noise ratio or signal level (dBm) before attempting to trigger forced client roaming'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'roam_trigger_interval', _('roam_trigger_interval'), _('Minimum time (ms) between client roaming trigger attempts'));
+	o.optional    = true;
+	o.placeholder = 60000;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'roam_kick_delay', _('roam_kick_delay'), _('Timeout (in 100ms beacon intervals) for client roam requests'));
+	o.optional    = true;
+	o.placeholder = 100;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'signal_diff_threshold', _('signal_diff_threshold'), _('Minimum signal strength difference until AP steering policy is active'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'initial_connect_delay', _('initial_connect_delay'), _('Initial delay (ms) before responding to probe requests (to allow other APs to see packets as well)'));
+	o.optional    = true;
+	o.placeholder = 0;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Flag, 'load_kick_enabled', _('load_kick_enabled'), _('Enable kicking client on excessive channel load (0/1)'));
+	o.optional    = true;
+
+	o = s.option(form.Value, 'load_kick_threshold', _('load_kick_threshold'), _('Minimum channel load (%) before kicking clients'));
+	o.optional    = true;
+	o.placeholder = 75;
+	o.datatype    = 'uinteger';
+	
+	o = s.option(form.Value, 'load_kick_delay', _('load_kick_delay'), _('Minimum amount of time (ms) that channel load is above threshold before starting to kick clients'));
+	o.optional    = true;
+	o.placeholder = 10000;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'load_kick_min_clients', _('load_kick_min_clients'), _('Minimum number of connected clients before kicking based on channel load'));
+	o.optional    = true;
+	o.placeholder = 10;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'load_kick_reason_code', _('load_kick_reason_code'), _('Reason code on client kick based on channel load (default: WLAN_REASON_DISASSOC_AP_BUSY)'));
+	o.optional    = true;
+	o.placeholder = 5;
+	o.datatype    = 'uinteger';
+
+	o = s.option(form.Value, 'node_up_script', _('node_up_script'), _('Script to run after bringing up a node'));
+	o.optional    = true;
+	o.datatype    = 'string';
+
+	o = s.option(form.Value, 'event_log_types', _('event_log_types'), _('Message types to include in log. Available types: probe_req_accept probe_req_deny, auth_req_accept, auth_req_deny, assoc_req_accept, assoc_req_deny, load_kick_trigger, load_kick_reset, load_kick_min_clients, load_kick_no_client, load_kick_client, signal_kick'));
+	o.optional    = true;
+	o.datatype    = 'list(string)';
+
+	o = s.option(form.Value, 'ssid_list', _('ssid_list'), _('List of SSIDs to enable steering on'));
```

Comment:
This must be a dynamic list I suppose.
```suggestion
	o = s.option(form.DynamicList, 'ssid_list', _('ssid_list'), _('List of SSIDs to enable steering on'));
```

Reply:
good suggestion